### PR TITLE
QOL: Allow userfraction of 1.0

### PR DESCRIPTION
**Pull request by MartenM**
Allows a userFraction of 1.0 to be input by a user. Anything higher than 0 is still accounted for as invalid.
Changes are currently not tested.

Another solution would be to change this at the start of the program when reading the inputs, however, that might cause confusion when debugging and the user sees the userFraction as 'undefined'.

### CODE REVIEW

File: src/edits.ts [hunk 1]
```diff
@@ -178,6 +178,12 @@ async function addReleasesToTrack(appEditId: string, options: EditOptions, versi
     core.debug(`track=${options.track}`)
     if (options.userFraction) {
         core.debug(`userFraction=${options.userFraction}`)
+
+        // When all users are targeted (i.e. userFraction equal to 1) modify it to be undfined as required by the update
+        // function provided by androidPublisher.edit.tracks
+        if (options.userFraction === 1.0) {
+            options.userFraction = undefined;
```
Comment (boswelja) [inline src/edits.ts:185]: Please at least add a warning that userFraction is ignored. I don't think we want people getting into a habit of passing data that won't be used

Comment (boswelja) [reply src/edits.ts:185]: It should also only be a warning for "completed" releases. We still want to fail on invalid config (partial rollout at 100% is invalid)

**Review by boswelja — CHANGES_REQUESTED** [review]

**Comment by boswelja** [comment]
Hey @MartenM, are you still interested in working on this?

**Comment by MartenM** [comment]
Ah sorry, I missed the feedback. Will check if I have some time later this week.

**Comment by boswelja** [comment]
Hey @MartenM, did you get a chance to look at this again?

**Comment by MartenM** [comment]
I have started working on my Masters's Thesis so my spare time has been limited quite a bit.
Feel free to close this draft if you want.

**Comment by boswelja** [comment]
No worries, thanks for letting me know!

**Pull request closed without merging**